src/main/java/me/clip/placeholderapi/PlaceholderAPI.java

```diff
@@ -374,18 +398,18 @@ public static boolean unregisterPlaceholderHook(Plugin plugin) {
   }
 
   public static String setPlaceholders(Player p, String text) {
-    return setPlaceholders((OfflinePlayer) p, text, PLACEHOLDER_PATTERN);
+    return setPlaceholders(p, text, PLACEHOLDER_PATTERN);
```

Comment:
You sure this won't cause issues?

Reply:
Player extends OfflinePlayer so there shouldn't be any problems, the cast is redundant.

I'll triple check everything works when I get time to today.

Reply:
Everything seems to be working fine, OtherPlayer and Player placeholders function as they should.